src/vm/excep.cpp

```diff
@@ -2428,9 +2428,7 @@ void StackTraceInfo::SaveStackTrace(BOOL bAllowAllocMem, OBJECTHANDLE hThrowable
                         }
                     }
 
-                    if (bSkipLastElement && gc.stackTrace.Size() != 0)
-                        gc.stackTrace.AppendSkipLast(m_pStackTrace, m_pStackTrace + m_dFrameCount);
-                    else
+                    if (!bSkipLastElement)
```

Comment:
Despite reading the code, I cannot form a picture of what is going on here. What is the purpose of `bSkipLastElement` ?

Reply:
From my analysis, this flag is set during a rethrown exception handling.
 https://github.com/dotnet/coreclr/blob/591459f32b7104a1d5c37e1a648490c8e2eb15a4/src/vm/exceptionhandling.cpp#L2555-L2558